content/en/platform/corda/4.11/enterprise/two-phase-finality.md

```diff
@@ -0,0 +1,69 @@
+---
+date: '2023-03-30T12:00:00Z'
+menu:
+  corda-enterprise-4-11:
+    parent: corda-enterprise-4-11-cordapps-flows
+tags:
+- two
+- phase
+- finality
+title: Two Phase Finality
+weight: 40
+---
+
+## Introduction
+
+Finality refers to the act of notarizing, recording, and sharing a transaction with all of its participants. Finality enables ledger consistency.
+
+An **initiator** uses the built-in flow called `FinalityFlow` to finalize a transaction:
+
+1. Send the transaction to the chosen notary and, if necessary, satisfy the notary that the transaction is valid.
+2. Record the transaction in the local vault, if it is relevant (that is, it involves the owner of the node).
+3. Send the fully signed transaction to the other participants for recording also.
+
+One or more **receivers** use the built-in flow `ReceiveFinalityFlow` to receive and record the finalized transaction.
+
+## Conventional implementation
+
+Up until Corda 4.10, the finality flow protocol was implemented using a single pass transaction sharing mechanism as depicted below:
+
+{{< figure alt="Conventional Single Phase Finality Protocol" width=100% zoom="./resources/C4-finality-conventional.png" >}}
+
+{{< note >}}
+* Peers only receive the finalized transaction after successful notarization.

```

Comment:
```suggestion
* Peers only received the finalized transaction after successful notarization.

```

Reply:
Not sure this is the right tense - the second would be used if talking about a single event in the past; I think the present conditional (sic?) works better here.